src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -967,6 +967,13 @@ std::string ContextAttribute::check
   size_t len;
   char errorMsg[128];
 
+  if (((ciP->apiVersion == "v2") && (len = strlen(name.c_str())) < MIN_ID_LEN) && (requestType != EntityAttributeValueRequest))
+  {
+    snprintf(errorMsg, sizeof errorMsg, "attribute name length: %zd, min length supported: %d", len, MIN_ID_LEN);
+    alarmMgr.badInput(clientIp, errorMsg);
+    return std::string(errorMsg);
+  }
```

Comment:
Is this check not implemented elsewhere?
If it is, that part should be removed now that the check is done here 'instead'.